docs/get-started/roles-and-permissions/lbac.md

```diff
@@ -0,0 +1,41 @@
+# Labels for access control in PMM
+
+
+Label-based access control in PMM allows you to manage who has access to metrics based on labels. By creating roles, you can specify which data can be queried based on specific label criteria, for instance, allowing the QA team to view data related to test environments. 
+ 
+With Label-based access control, you can associate multiple labels with a role, ensuring only data from series that match your defined labels is returned. 
+
+
+## Predefined vs Custom labels
```

Comment:
The wording should be `Standard` instead of `Predefined` right?

https://github.com/percona/pmm-doc/pull/995#discussion_r1111899574

Reply:
its Predefined as per the doc.